artifacts/devnet-1/early-adoption/sources/testing.csv

```diff
@@ -0,0 +1,10 @@
+0xC234dBA03943C9238067cDfBC2761844133DD386
```

Comment:
What is the purpose of the file?

Reply:
this file is not script-generated, just source
to keep the same structure everywhere